### PR TITLE
chore(time-travel): added new floors and rooms

### PR DESCRIPTION
**Pull request by Karguel1995**
Hello! 
It's my first pull request. Hope everything is fine, and you like it :)

### CODE REVIEW

File: english/time-travel/0/7.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**You are looking around and notice strange-looking mushrooms 🍄.**
+
+When you look at mushrooms, you hear animal sounds.
+
+- [🍄 Eat mushrooms](7-1.md)
```
Comment (hkumar0132) [inline english/time-travel/0/7.md:5]: When I click on 'Eat Mushrooms', I am redirected to 7-1.md.
These 2 feels disconnected to me as there is no mention about eating mushrooms.

**Review by hkumar0132 — CHANGES_REQUESTED** [review]
Please make these changes

File: english/time-travel/0/1.md [hunk 2]
```diff
@@ -5,4 +5,6 @@
 You are thrown out of it while travelling and your body is torn up 🩸.
 
 You. Are. Dead.
-💀☠😐⚰
\ No newline at end of file
+💀☠😐⚰
+
+[Restart the game](../begin-journey.md)
```
[the remark this answers is not in the document]
Comment (Karguel1995) [reply english/time-travel/0/1.md:10]: I will change this

File: english/time-travel/0/7.md (hunk 1 above)
Comment (Karguel1995) [reply english/time-travel/0/7.md:5]: Here I meant that after you ate mushrooms and world started to spinning and you are high

**Comment by Karguel1995** [comment]
Upload!
@hkumar0132 , could you check?

File: english/time-travel/0/7-1.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+**The whole world starts spinning 🌀 and you fall to the ground.**
```
Comment (hkumar0132) [inline english/time-travel/0/7-1.md]: Please change it to :
**You feel dizzy from eating mushrooms. Your world starts spinning 🌀 and you fall to the ground.**

**Comment by hkumar0132** [comment]
> Upload!
> @hkumar0132 , could you check?

Appreciate the efforts!
It seems like you have made a merge commit. 

@ming-tsai  Is it okay to approve a 'merge commit'?

**Comment by ming-tsai** [comment]
> > Upload!
> > @hkumar0132 , could you check?
> 
> Appreciate the efforts!
> It seems like you have made a merge commit. 
> 
> @ming-tsai  Is it okay to approve a 'merge commit'?

DO NOT merge the pull request, only approve it, thanks

File: english/time-travel/0/7-1.md (hunk 3 above)
Comment (ming-tsai) [reply english/time-travel/0/7-1.md]: And should add an option or jump into WIP.md

**Comment by hkumar0132** [comment]
> > > Upload!
> > > @hkumar0132 , could you check?
> > 
> > 
> > Appreciate the efforts!
> > It seems like you have made a merge commit.
> > @ming-tsai  Is it okay to approve a 'merge commit'?
> 
> DO NOT merge the pull request, only approve it, thanks

Great! Waiting for the changes from @Karguel1995

File: english/time-travel/0/7-1.md (hunk 3 above)
Comment (Karguel1995) [reply english/time-travel/0/7-1.md]: Thanks!
What do you mean by WIP.md?

Comment (ming-tsai) [reply english/time-travel/0/7-1.md]: Ok, sorry in this dungeon doesn't have `WIP.md`, so please add option to restart the game

**Review by hkumar0132 — APPROVED** [review]

**Pull request merged**